content/pages/how_you_can_use_python.md

```diff
@@ -0,0 +1,29 @@
+Title: {{ How You Can Use Python }}
+Content Date: 2018-10-23 22:27
+
+Tags: python, learning, programming, projects, ideas
+
+Authors: Cleveland PyLadies, Daniel Medina
+
+Summary:  A brief summary of how you can use and apply Python to solve real-world problems.
+
+Python is an extremely versatile programming language known for its beginner friendly syntax.  It is used by NASA, Google, Dropbox, Quora, and many others.
+
+Python is primarily used for scripting, the automation of short repetitive tasks.
+
+Here are some of the ways that Python is used across multiple areas:
+
+  Science with [SciPy](https://www.datacamp.com/community/tutorials/python-scipy-tutorial), [NumPy](https://www.machinelearningplus.com/python/numpy-tutorial-part1-array-python-examples/)
```

Comment:
Could I ask you to reformat this section? I like that you call out specific domains, but Web Dev is between Science and ML. Those two have more overlap than web dev and either. Also, use the phrase "Scientific Computing" rather than "Science."

Since there are SO many more libraries that are in heavy rotation among each of the domains you're covering, it might be best to frame this more explicitly as examples.

Reply:
This is more of a library-based summary, which I'm totally ok with. We just need to make that clear in the wording.

Reply:
Maybe add bullet points to the domains? `* Scientific Computing - many scientists use [SciPy]...`